module-2/backend/week_two.md

```diff
@@ -5,33 +5,77 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+* ORM - allows users to interact with the database easier
 2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+* class methods. UAgent has many payloads, Payload belongs_to UAgent
```

Comment:
Good example! I think the point we were trying to make is that you'll need a specific instance of UAgent to use `.payloads` on it.